charts/opensearch/CHANGELOG.md

```diff
@@ -12,9 +12,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 ### Fixed
 ### Security
-
 ---
-
+## [2.19.0]
```

Comment:
Same can you update the change log at bottom of this file?

Reply:
Done